aodndata/templates/soop_tmv_nrt_nc_template_mooring.json

```diff
@@ -93,6 +94,7 @@
       "_dimensions": ["TIME"],
       "_datatype": "float64",
       "long_name": "mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water_concentration_of_chlorophyll_in_sea_water",
```

Comment:
didn't you mean "mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water" ?

Reply:
absolutely